test/integration/IntegrationTest.h

```diff
@@ -36,3 +36,100 @@ struct IntegrationTest : public ::testing::Test
 
     void initBridge(config::Config& config);
 };
+
+template <typename T>
+class GroupCall
+{
+public:
+    GroupCall(const std::vector<T*> clients) : _clients(clients) {}
```

Comment:
```suggestion
    GroupCall(const std::vector<T*>& clients) : _clients(clients) {}    
```
--

I know this is a test code only but maybe it would be even better use `std::initializer_list` to avoid the temporary heap allocation created by the temporary `std::vector`. like:
```suggestion
    GroupCall(std::initializer_list<T*> clients) : _clients(clients) {}
```
And you give the braced initialization, like:
`GroupCall<SfuClient<ColibriChannel>> groupCall = {&client1, &client2, &client3};`